app/controllers/duffel_items_controller.rb

```diff
@@ -10,4 +10,19 @@ def create
   def index
     @items = @duffel.duffel_items
   end
+
+  def update
+    # binding.pry
+    function = params[:function]
+    if function == "add"
+      session[:duffel][params[:id]] += 1
+    else
+      session[:duffel][params[:id]] -= 1
+      if session[:duffel][params[:id]] == 0
+        session[:duffel] = session[:duffel].except(params[:id])
+        @duffel.contents = session[:duffel]
+      end
+    end
+    redirect_to '/duffel'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.